src/middlewared/middlewared/plugins/cloud_sync.py

```diff
@@ -144,17 +147,32 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 async def rclone_check_progress(job, proc):
+    dropbox__restricted_content = False
     while True:
         read = (await proc.stdout.readline()).decode()
-        job.logs_fd.write(read.encode("utf-8", "ignore"))
         if read == "":
             break
+        if "failed to open source object: path/restricted_content/" in read:
+            job.internal_data["dropbox__restricted_content"] = True
+            dropbox__restricted_content = True
+        job.logs_fd.write(read.encode("utf-8", "ignore"))
         reg = RE_TRANSF.search(read)
         if reg:
             transferred = reg.group(1).strip()
             if not transferred.isdigit():
                 job.set_progress(None, transferred)
 
+    if dropbox__restricted_content:
+        message = "\n" + (
+            "We have detected that your DropBox sync failed due to restricted content being present in one of your\n"
+            "folders. It can be copyrighted content or just default DropBox manual pdf you get in your home directory\n"
+            "after signing up. All your other files were synchronized, but no deletions were performed as\n"
+            "synchronization is considered unsuccessful. Please inspect logs to determine which files are considered\n"
+            "restricted and exclude them from your synchronization. Refer to corresponding User Manual section for\n"
```

Comment:
> Refer to corresponding User Manual section for ...

Is it possible to insert a link here to that section in the guide?

Reply:
@a-stjohn that section of the guide is not written yet